spec/features/facets_spec.rb

```diff
@@ -29,9 +29,29 @@
     expect(page).to have_selector '.facet-values li:first', text: "Accident insurance"
     expect(page).to have_link "Numerical Sort"
     expect(page).to have_selector '.sort_options .active', text: "A-Z Sort"
-    
-    
   end
+  
+  it "should be able to sort more facet window by letter" do
+    visit catalog_facet_path("subject_topic_facet")
+    within ".modal-footer" do
+      click_on "A-Z Sort"
+    end
+    expect(page).to have_selector '.facet-values li:first', text: "Accident insurance"
+    expect(page).to have_css('.facet-values li',count: 20)
```

Comment:
Can you add a space after the commas here?

Can you also pick whether to use parenthesis around arguemtns or not, and stick to it?